frontend/src/Onboarding/LoginScreen.tsx

```diff
@@ -191,7 +199,7 @@ const LoginScreenComponent: React.FC<Props> = props => {
         error={
           (textFieldStr.length === 0 && beenEdited) ||
           !state.validEmail ||
-          !!state.error
+          state.error !== undefined
```

Comment:
nit maybe we can just do state.error == null in case if error is either null or undefined